test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1520,8 +1521,15 @@ UCS_TEST_P(test_ucp_am_nbx_dts, short_bcopy_send, "ZCOPY_THRESH=-1",
     });
 }
 
-UCS_TEST_P(test_ucp_am_nbx_dts, zcopy_send, "ZCOPY_THRESH=1", "RNDV_THRESH=-1")
+UCS_TEST_SKIP_COND_P(test_ucp_am_nbx_dts, zcopy_send,
+                     !is_proto_disabled() &&
+                             has_any_transport({"ud_v", "ud_x"}),
+                     "ZCOPY_THRESH=1", "RNDV_THRESH=-1")
 {
+    if (!am_lane_has_caps(UCT_IFACE_FLAG_AM_ZCOPY)) {
+        UCS_TEST_SKIP_R("am_zcopy is not supported");
+    }
+
```

Comment:
move to another pr

Reply:
Moved to https://github.com/openucx/ucx/pull/9215